aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_metric_attribute_generator.py

```diff
@@ -373,4 +373,5 @@ def _set_span_kind_for_dependency(span: ReadableSpan, attributes: BoundedAttribu
 
 def _log_unknown_attribute(attribute_key: str, span: ReadableSpan) -> None:
     message: str = "No valid %s value found for %s span %s"
-    _logger.log(DEBUG, message, attribute_key, span.kind.name, str(span.context.span_id))
+    if _logger.isEnabledFor(DEBUG):
+        _logger.log(DEBUG, message, attribute_key, span.kind.name, str(span.context.span_id))
```

Comment:
Is this the only place we log a message in the entire distro? If there are other places, we should change those too.

Reply:
Found a few more instances: https://github.com/search?q=repo%3Aaws-observability%2Faws-otel-python-instrumentation+_logger.debug&type=code

Reply:
I can follow up on these later.  Will cut a backlog item.